model/datapreprocess.py

```diff
@@ -4,6 +4,7 @@
 import os
```

Comment:
as we found that the feature extraction is not that slow, we decide not to use this datapreprocess. Please remove it